tlatools/org.lamport.tlatools/test/tla2sany/TestBuiltInOperatorInitialization.java

```diff
@@ -0,0 +1,78 @@
+package tla2sany;
+
+import java.util.Arrays;
+import java.util.Enumeration;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+import tla2sany.semantic.ASTConstants;
+import tla2sany.semantic.BuiltInOperators;
+import tla2sany.semantic.BuiltInOperators.BuiltInOperator;
+import tla2sany.semantic.Context;
+import tla2sany.semantic.OpDefNode;
+import tla2sany.semantic.SymbolNode;
```

Comment:
This test references classes from tla2sany.semantic. If you moved it to the tla2sany.semantic package, you would not need to change Context.Pair visibility from package private to public.

Reply:
I do think sorting the `tla2sany` tests into submodules modules in this way is a good idea though, so I added another commit which makes this change and also restricts the visibility of `Context.Pair` again.